projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -449,7 +449,7 @@ public function post_page_metabox() {
 				$labels = array();
 
 				foreach ( $connections_data as $connection_data ) {
-					if ( ! $connection_data['enabled'] || ( isset( $connection_data['is_healthy'] ) && false === $connection_data['is_healthy'] ) ) {
+					if ( ! $connection_data['enabled'] ) {
```

Comment:
Should we be checking that the status is ok too?

Reply:
`status` by default is `null` and is now set only on connection test results.

Reply:
Right. Do you think it could be safer to check status here too? I feel like we should be storing that, so we'll just have to put this check back in when we do.

Reply:
Yeah, we can put that that back when it's implemented.